src/Point/Point.cs

```diff
@@ -19,7 +19,7 @@ namespace EulynxLive.Point
 {
     public partial class Point : BackgroundService, IPoint
     {
-        public IPointToInterlockingConnection Connection { get; }
+        public IPointToInterlockingConnection Connection { get; set; }

```

Comment:
I think I don't like that we can re-connect a closed connection object. I think it would be cleaner to request a new connection from the connection provider. (At least we should make this setter private.)